### ISSUE TITLE
duplicate interface definition for class 'RCTConvert', redefinition of 'RCTLogLevel'

### ISSUE THREAD

**Issue by jasonrobinson** [react-native-community/react-native-svg#214]
First, great job on the library!

Things were working well on RN@0.39, but on a fresh install of RN@0.40.0 + RNSVG, the iOS build fails with errors around `error: duplicate interface definition for class 'RCTConvert'`, with `error: redefinition of 'RCTLogLevel'` and associated items. 

Here is a fresh RN0.40.0 repo, https://github.com/jasonrobinson/svgtest , with the latest RNSVG (currently 4.4.1) linked via `react-native link react-native-svg`. 

The relevant build output is [build_output.txt](https://github.com/react-native-community/react-native-svg/files/685397/build_output.txt). The build succeeds upon downgrading the repo that I linked to RN@0.39, so this seems to be RN@0.40.0 specific. Bumping react to 15.4.1 in `package.json` does not seem to help.

**Comment by magicismight** [comment]
Just released 4.5.0 which is supported for react-native@0.40.0

**Comment by jasonrobinson** [comment]
Thanks @magicismight -- that installs well on a new repo. 

However, I think that the larger problem that I had was some slightly outdated dependencies, which led to issues building after running pretty much any `react-native link <somepackage>`. After updating RNCLI and making sure that `package.json` is up to date vis-a-vis a clean repo, things compile correctly again. I should have checked that earlier. 

So, apologies for bringing up the issue within this repo. 

**tl;dr** for anyone else who has issues using `react-native link <somepackage>` after a react-native upgrade using any of the convenience tools, check all dependencies, dev dependencies, and global items (e.g. RNCLI) versus a clean repo.
